insights/parsers/yum_updateinfo.py

```diff
@@ -0,0 +1,21 @@
+"""
+UpdateInfo - updateinfo(available advisories) provided by running `yum updateinfo list`
+"""
```

Comment:
@semtexzv please review the [documentation guidelines](https://insights-core.readthedocs.io/en/latest/docs_guidelines.html) for required documentation of components.